src/main/kotlin/uk/gov/dluhc/printapi/config/ElectorDocumentPdfTemplateProperties.kt

```diff
@@ -5,7 +5,10 @@ import org.springframework.boot.context.properties.ConstructorBinding
 
 @ConfigurationProperties(prefix = "temporary-certificate.certificate-pdf")
 @ConstructorBinding
-data class TemporaryCertificatePdfTemplateProperties(
+class TemporaryCertificatePdfTemplateProperties(english: English, welsh: Welsh) :
+    ElectorDocumentPdfTemplateProperties(english, welsh)
+
+abstract class ElectorDocumentPdfTemplateProperties(
```

Comment:
Came to a [general conscientious on slack](https://valtechhub.slack.com/archives/C03G86348N9/p1676452560461249) that the common term for the 3 types of certificate document (VAC, Temp Cert and AED) is "Elector Document"